src/pages/ProfilePage/ProfileComponent/HeaderComponent.js

```diff
@@ -0,0 +1,71 @@
+import { Avatar, Box, Button, makeStyles, Typography } from "@material-ui/core";
+import React from "react";
+
+const useStyles = makeStyles((theme) => ({
+  profiletitle: {
+    padding: `${theme.spacing(2)}px`,
+    "& .MuiTypography-root": {
+      fontSize: "1.3rem",
+      fontWeight: 400,
+    },
+  },
+  Avatar: {
+    width: "70px",
+    height: "70px",
+  },
+  headerContainer: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    justifyContent: "flex-start",
+    marginLeft: `${theme.spacing(2)}px`,
+    color: "#000",
+  },
+  studentdetails: {
+    marginLeft: "10px",
+    padding: `${theme.spacing(1)}px`,
+    "& .MuiTypography-h2": {
+      fontSize: "15px",
+      fontWeight: 400,
+      marginBottom: `${theme.spacing(0.5)}px`,
+    },
+    "& .MuiTypography-h3": {
+      fontSize: "12px",
+      fontWeight: 400,
+      marginBottom: `${theme.spacing(0.5)}px`,
+    },
+  },
+  Editbutton: {
+    marginTop: "10px",
+    background: "#fff",
+    textTransform: "none",
+    borderRadius: "30px",
+    border: "1px solid black",
+    padding: "0 5px 0 5px",
```

Comment:
Add left and right internal padding as like you have did in design.
Add values in multiples of 8.